scripts/eval_privacy_conformance.py

```
@@ -0,0 +1,138 @@
+) -> int:
+    keep_artifacts = os.environ.get("PRIVACY_EVAL_KEEP_ARTIFACTS") == "1"
+    artifact_directory = Path(tempfile.mkdtemp(prefix="privacy-eval-"))
+    output_path = artifact_directory / "results.jsonl"
+    environment = os.environ.copy()
+    environment["PROMPTFOO_DISABLE_TELEMETRY"] = "1"
+    environment["PROMPTFOO_CONFIG_DIR"] = str(
+        artifact_directory / "promptfoo-config"
+    )
+    environment["PYTHONPATH"] = str(REPOSITORY_ROOT / "backend")
+    try:
+        promptfoo = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)